util.go

```diff
@@ -4,8 +4,15 @@ package gosnowflake
 
```

Comment:
Have you considered adding this functionality to dsn.go? I'm not sure where it belongs better, but `util` is like a generic bag for everything and it usually grows into huge mess.

Reply:
Great idea. I have moved the functionality to the `dsn.go`.